packages/itwin/grouping-mapping-widget/src/widget/components/SelectIModel.scss

```diff
@@ -4,18 +4,14 @@
 *--------------------------------------------------------------------------------------------*/
 @import '~@itwin/itwinui-css/scss/variables';
 
-.imodel-grid-container {
+.gmw-imodel-grid-container {
   display: flex;
   flex-direction: column;
   flex-grow: 1;
   min-height: 0;
 
-  .imodel-grid {
+  .gmw-imodel-grid {
     overflow: auto;
     flex-grow: 1;
-
-    .search-input {
```

Comment:
Does removal of the `.search-input` style cause any UI change? Could you provide before/after screenshots of the iModel grid?

Reply:
![With search-input](https://user-images.githubusercontent.com/87046458/185710573-79090bfc-38e1-42ea-8192-e9288c23771d.jpeg)

![Without search-input](https://user-images.githubusercontent.com/87046458/185710574-208a95e4-d211-42ac-bc07-765a1fb96c8c.jpeg)

I do not think there are any changes, but I may not be looking at the right things